### ISSUE TITLE
Ubuntu 18.04,The provider 'virtualbox' that was requested to back the machine 'default' is reporting that it isn't usable on this system

### ISSUE THREAD

**Issue by MilenkoM** [hashicorp/vagrant#10764]
Please note that the Vagrant issue tracker is in priority reserved for bug reports and enhancements. For general usage questions, please use the Vagrant mailing list:
https://groups.google.com/forum/#!forum/vagrant-up. Thank you!

**Tip:** Before submitting your issue, don't hesitate to remove the above introductory text, possible empty sections (e.g. References), and this tip.

### Vagrant version
Run `vagrant -v` to show the version. If you are not running the latest version
of Vagrant, please upgrade before submitting an issue.
```
vagrant -v
Vagrant 2.2.0
```

### Host operating system
This is the operating system that you run locally.
Ubuntu 18.04
### Guest operating system
This is the operating system you run in the virtual machine.
Virtual Box
6.0.4r128413
### Vagrantfile
```ruby
# Copy-paste your Vagrantfile here (but don't include sensitive information such as passwords, authentication tokens, or email addresses)
```

Vagrant.configure("2") do |config|

  config.vm.box = "ubuntu/xenial64"

Please note, if you are using Homestead or a different Vagrantfile format, we
may be unable to assist with your issue. Try to reproduce the issue using a
vanilla Vagrantfile first.

### Debug output
Provide a link to a GitHub Gist containing the complete debug output:
https://www.vagrantup.com/docs/other/debugging.html. The debug output should
be very long. Do NOT paste the debug output in the issue, just paste the
link to the Gist.
 vagrant up
Bringing machine 'default' up with 'libvirt' provider...
==> default: Box 'ubuntu/xenial64' could not be found. Attempting to find and install...
    default: Box Provider: libvirt
    default: Box Version: >= 0
==> default: Loading metadata for box 'ubuntu/xenial64'
    default: URL: https://vagrantcloud.com/ubuntu/xenial64
The box you're attempting to add doesn't support the provider
you requested. Please find an alternate box or use an alternate
provider. Double-check your requested provider to verify you didn't
simply misspell it.

### Expected behavior
What should have happened?
I expected new VBox creation.

### Actual behavior
What actually happened?
The box you're attempting to add doesn't support the provider
you requested. Please find an alternate box or use an alternate
provider. Double-check your requested provider to verify you didn't
simply misspell it.

If you're adding a box from HashiCorp's Vagrant Cloud, make sure the box is
released.

Name: ubuntu/xenial64
Address: https://vagrantcloud.com/ubuntu/xenial64
Requested provider: [:libvirt]

What is important is that everything works fine with older Vbox 5.2(on my laptop)
### Steps to reproduce
1. vagrant init ubuntu/xenial64
2.vagrant up
3.

### References
Are there any other GitHub issues (open or closed) that should be linked here?
For example:
- GH-1234
- ...

**Comment by briancain** [comment]
Hey there @MilenkoM - do you have the `libvirt` provider installed?

**Comment by MilenkoM** [comment]
I think so
```
dpkg -s qemu-kvm | grep Version
Version: 1:2.11+dfsg-1ubuntu7.12
```

**Comment by briancain** [comment]
What is the result of `vagrant plugin list`?

**Comment by MilenkoM** [comment]
```
vagrant plugin list
==> vagrant: A new version of Vagrant is available: 2.2.4 (installed version: 2.2.0)!
==> vagrant: To upgrade visit: https://www.vagrantup.com/downloads.html

vagrant-libvirt (0.0.45, global)
```

**Comment by briancain** [comment]
@MilenkoM - I believe if you have the `vagrant-libvirt` plugin installed, it ends up being the default provider. I recommend either adding this line to your Vagrantfile, or using the CLI flag `--provider virtualbox` when you bring up the guest.

```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "ubuntu/xenial64"
  config.vm.provider :virtualbox
end
```

But this is behaving as expected with the libvirt plugin installed. Hopefully that makes things work for you. Thanks!
